platform/security/idp/security-idp-server/src/test/resources/saml.xml

```diff
@@ -93,7 +93,7 @@
             <saml2:Audience>https://localhost:8993/services</saml2:Audience>
         </saml2:AudienceRestriction>
     </saml2:Conditions>
-    <saml2:AuthnStatement AuthnInstant="2013-04-23T23:39:54.777Z">
+    <saml2:AuthnStatement AuthnInstant="2013-04-23T23:39:54.777Z" SessionIndex="1">
```

Comment:
❗️ Based on the schema, it looks like `SessionIndex` is optional, so we should have tests with an `Assertion` that has an `AuthnStatement` without a `SessionIndex`.

Reply:
There is `LogoutRequestServiceTest.testGetLogoutRequestNoSessionIndex`. It isn't working the way I thought it was, so I'm pushing a fix for it soon.

Reply:
Fixed